messages/teleporter/message.go

```diff
@@ -65,16 +67,26 @@ func UnpackTeleporterMessage(messageBytes []byte) (*TeleporterMessage, error) {
 	return &teleporterMessage.TeleporterMessage, nil
 }
 
-func PackReceiverMessage(inputStruct ReceiveCrossChainMessageInput) ([]byte, error) {
+// PackReceiveCrossChainMessage packs a ReceiveCrossChainMessageInput to form a call to the receiveCrossChainMessage function
+func PackReceiveCrossChainMessage(inputStruct ReceiveCrossChainMessageInput) ([]byte, error) {
 	return EVMTeleporterContractABI.Pack("receiveCrossChainMessage", inputStruct.RelayerRewardAddress)
 }
 
-func PackMessageReceivedMessage(inputStruct MessageReceivedInput) ([]byte, error) {
+// PackMessageReceived packs a MessageReceivedInput to form a call to the messageReceived function
+func PackMessageReceived(inputStruct MessageReceivedInput) ([]byte, error) {
 	return EVMTeleporterContractABI.Pack("messageReceived", inputStruct.OriginChainID, inputStruct.MessageID)
 }
 
+// UnpackMessageReceivedResult attempts to unpack result bytes to a bool indicating whether the message was received
 func UnpackMessageReceivedResult(result []byte) (bool, error) {
 	var success bool
 	err := EVMTeleporterContractABI.UnpackIntoInterface(&success, "messageReceived", result)
 	return success, err
 }
+
+// PackSendCrossChainMessageEvent packs the SendCrossChainMessage event type. PackEvent is documented as not supporting struct types, so this should be used
+// with caution. Here, we only use it for testing purposes. In a real setting, the Teleporter contract should pack the event.
```

Comment:
Can we make this warning a little more obvious? Maybes start one line of the comment with "CAUTION". I would totally miss this most of the time.

Reply:
Good call, clarified that. I also thought about moving it to `package tests`, but that unfortunately creates an import cycle. If we decide to move these Teleporter packing/unpacking utils to `teleporter`, we can spend more energy putting up better guardrails, but IMO a more explicit comment is sufficient for now.